src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -248,8 +248,9 @@ def get_yaml_template(cmd_value, timeout, file):
     :param str timeout: The timeout for each step
     :param str file: The task definition
     """
-    yaml_template = "version: v1.1.0\n"
+    yaml_template = ""
     if cmd_value:
+        yaml_template += "version: v1.1.0\n"
```

Comment:
Can you add a comment to call out it needs to append the version for cmd source? For file source, we expect it includes the version in the file.

Reply:
added comments